src/Symfony/Component/Form/Resources/translations/validators.fa.xlf

```diff
@@ -4,15 +4,15 @@
         <body>
             <trans-unit id="28">
                 <source>This form should not contain extra fields.</source>
-                <target>این فرم نباید فیلد اضافی داشته باشد.</target>
+                <target>این فرم نباید شامل فیلد اضافه ای باشد.</target>
             </trans-unit>
             <trans-unit id="29">
                 <source>The uploaded file was too large. Please try to upload a smaller file.</source>
-                <target>فایل بارگذاری شده بسیار بزرگ است. لطفا فایل کوچکتری را بارگزاری کنید.</target>
+                <target>فایل بارگذاری شده بسیار بزرگ می باشد. لطفا فایل کوچکتری را بارگذاری نمایید.</target>
```

Comment:
Google also recommends some changes here:

```
فایل بارگذاری شده بسیار بزرگ میباشد. لطفا فایل کوچکتری را بارگذاری نمایید.
```

Reply:
This word must be written separately:
**می باشد**

And the word for the **uploaded** is as follows:
**بارگذاری**

Reply:
OK, then it's a "false positive" in Google Translate. Thanks for double checking it.

Reply:
Thank you very much for your review.
Yes, Google has a lot of false translations, and I'm trying to improve it.
I have contributed a lot to Google in the native language of Persian since 2015.